app.go

```diff
@@ -25,15 +25,33 @@ func app(cmd *cobra.Command, args []string) {
 	var version string
 	var err error
 
-	// Get version of Docker benchmark to run
-	if dockerVersion != "" {
-		version = dockerVersion
+	// Benchmark flag is specify
+	if benchmarkVersion != "" {
+
+		// Check for not specify both --version and --benchmark
+		if dockerVersion != "" {
+			util.ExitWithError(
+				fmt.Errorf("Version check failed: Error can't specify both --version and --benchmark flags\nIf not specify neither docker version will be auto detect"))
```

Comment:
See https://github.com/aquasecurity/kube-bench/blob/714430c7fcb718de0b9d9d350aec26445d218b46/cmd/common.go#L279

```suggestion
				fmt.Errorf("It is an error to specify both --version and --benchmark flags"))
```